Wangscape/Options.h

```diff
@@ -17,13 +20,20 @@ class Options
     typedef std::vector<Clique> CliqueList;
 
     std::string filename;
+    std::string directory;
```

Comment:
I don't know either, which would be better in that case. You can make it standalone class for me :).